src/coreclr/jit/compiler.h

```diff
@@ -2522,6 +2522,22 @@ class Compiler
                                  unsigned    simdSize,
                                  bool        isSimdAsHWIntrinsic);
 
+    GenTree* gtNewSimdShuffleNode(Compiler*   comp,
+                                  var_types   type,
+                                  GenTree*    op1,
+                                  GenTree*    op2,
+                                  CorInfoType simdBaseJitType,
+                                  unsigned    simdSize,
+                                  bool        isSimdAsHWIntrinsic);
```

Comment:
Isn't this an instance method on `Compiler`? Isn't the first param is unnecessary?

Reply:
Yes. My brain was was apparently elsewhere thinking this was another function.

Reply:
Fixed